### PR TITLE
chore: add configs for support robots

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

File: .github/lock.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# Number of days of inactivity before a closed issue or pull request is locked
+daysUntilLock: 10
+
+# Skip issues and pull requests created before a given timestamp. Timestamp must
+# follow ISO 8601 (`YYYY-MM-DD`). Set to `false` to disable
+skipCreatedBefore: false
+
+# Issues and pull requests with these labels will be ignored. Set to `[]` to disable
+exemptLabels:
```
Comment (ushkal) [inline .github/lock.yml]: Shouldn't we add the next labels: 
discussion, enhancement, help wanted, more-information-needed, bug

File: .github/no-response.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
+
+# Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 10
+# Label requiring a response
+responseRequiredLabel: "STATE: Need clarification"
```
Comment (ushkal) [inline .github/no-response.yml]: 'more-information-needed' in our case

Comment (churkin) [reply .github/no-response.yml]: I'll rename this label to "STATE: Need clarification". Unfortunatly this label's name is hard coded in the robot ((

File: .github/lock.yml (hunk 1 above)
Comment (churkin) [reply .github/lock.yml]: Why? If the issue is closed we can lock it.

**Pull request merged**